src/pull.js

```diff
@@ -30,6 +30,35 @@ export default class Pull {
     this.side = 'remote'
   }
 
+  // Pull multiple files/dirs metadata at once, given their ids
+  async pullMany (ids: string[]) {
+    let failedIds = []
+
+    for (let id of ids) {
+      try {
+        await this.pullOne(id)
```

Comment:
Since I had no idea what was happening in prep/merge when I first wrote this code, I prefered not to do it in parallel, and keep it as a possible future optimization.

I can change it if you don't think it'll be an issue.